django_project/frontend/utils/data_table.py

```diff
@@ -177,6 +196,16 @@ def sampling_report(queryset: QuerySet, request) -> List:
         end_year = request.GET.get("end_year")
         filters["year__range"] = (start_year, end_year)
 
+    activity = request.GET.get("activity")
+    if activity:
+        activity = (
+            activity.replace("_", " ")
```

Comment:
Please can we just use the same name in the frontend instead, so we don't need to do this.